<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-11-15T09:45:21 IST" name="Default suite" finished-at="2023-11-15T09:45:37 IST" duration-ms="16120">
    <groups>
    </groups>
    <test started-at="2023-11-15T09:45:21 IST" name="Default test" finished-at="2023-11-15T09:45:37 IST" duration-ms="16120">
      <class name="testscripts.scenario1.TC_Fusion_ERP_AP_01_01">
        <test-method is-config="true" signature="suiteSetup(org.testng.ITestContext)[pri:0, instance:testscripts.scenario1.TC_Fusion_ERP_AP_01_01@3ce1e309]" started-at="2023-11-15T09:45:19 IST" name="suiteSetup" finished-at="2023-11-15T09:45:21 IST" duration-ms="1513" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@149debbb]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- suiteSetup -->
        <test-method is-config="true" signature="testMethodSetup()[pri:0, instance:testscripts.scenario1.TC_Fusion_ERP_AP_01_01@3ce1e309]" started-at="2023-11-15T09:45:21 IST" name="testMethodSetup" finished-at="2023-11-15T09:45:21 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMethodSetup -->
        <test-method signature="runTC4()[pri:0, instance:testscripts.scenario1.TC_Fusion_ERP_AP_01_01@3ce1e309]" started-at="2023-11-15T09:45:21 IST" name="runTC4" finished-at="2023-11-15T09:45:37 IST" duration-ms="15966" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Keyword createInvoice not found within any class inside the businesscomponents package]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Keyword createInvoice not found within any class inside the businesscomponents package
at org.testng.Assert.fail(Assert.java:99)
at supportlibraries.DriverScript.wrapUp(DriverScript.java:566)
at supportlibraries.DriverScript.driveTestExecution(DriverScript.java:104)
at testscripts.scenario1.TC_Fusion_ERP_AP_01_01.runTC4(TC_Fusion_ERP_AP_01_01.java:26)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runTC4 -->
        <test-method is-config="true" signature="testMethodTearDown()[pri:0, instance:testscripts.scenario1.TC_Fusion_ERP_AP_01_01@3ce1e309]" started-at="2023-11-15T09:45:37 IST" name="testMethodTearDown" finished-at="2023-11-15T09:45:37 IST" duration-ms="122" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMethodTearDown -->
        <test-method is-config="true" signature="suiteTearDown()[pri:0, instance:testscripts.scenario1.TC_Fusion_ERP_AP_01_01@3ce1e309]" started-at="2023-11-15T09:45:37 IST" name="suiteTearDown" finished-at="2023-11-15T09:45:37 IST" duration-ms="638" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- suiteTearDown -->
      </class> <!-- testscripts.scenario1.TC_Fusion_ERP_AP_01_01 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
